hlx_statics/scripts/lib-helix.js

```diff
@@ -283,7 +307,12 @@ export function decorateSections(main) {
       Object.keys(meta).forEach((key) => {
         if (key === 'style') {
           const styles = meta.style.split(',').map((style) => toClassName(style.trim()));
-          styles.forEach((style) => section.classList.add(style));
+          styles.forEach((style) => {
+            section.classList.add(style);
+            if (style === 'dark') {
```

Comment:
Not sure about this but looking at the rest of the code shouldn't we also test for a case where style== 'darkest' ?

Reply:
it seems to me that "lightest/darkest" is a block-specific setting and not a general style. I think we should stick to `dark` only, unless the lightest/darkest eventually gets promoted to a global style setting.